.github/workflows/runtests.yml

```diff
@@ -218,6 +222,33 @@ jobs:
       # Test modules with specific dependencies
       - run: bin/test_optional_dependencies.py
 
+  # -------------------- FLINT tests -------------------------------- #
+
+  python-flint:
+    needs: code-quality
+
+    # Currently wheels are available for python-flint only on Windows and OSX
+    # so we use OSX rather than Ubuntu for this job.
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - run: python -m pip install --upgrade pip
+      - run: pip install -r requirements-test.txt
+      - run: pip install python-flint
+      # Test modules that most directly use python-flint
+      - run: |
+          pytest --doctest-plus --doctest-glob='*.rst'    \
+                    sympy/polys                           \
+                    doc/src/modules/polys                 \
+                    sympy/ntheory                         \
+                    doc/src/modules/ntheory.rst           \
```

Comment:
I don't know if it's actually incorrect or not, but the extra backslash here is at least confusing the YAML syntax highlighting.

Reply:
It works okay in CI.

Actually I already changed this line in gh-25495 because I couldn't use `--doctest-plus` in the end (it needs better support for skipping doctests conditionally).